sentry_sdk/integrations/opentelemetry/potel_span_processor.py

```diff
@@ -135,15 +137,16 @@ def _root_span_to_transaction_event(self, span):
         if span.resource.attributes:
             contexts[OTEL_SENTRY_CONTEXT] = {"resource": dict(span.resource.attributes)}
 
-        event = {
-            "type": "transaction",
-            "transaction": description,
-            # TODO-neel-potel tx source based on integration
-            "transaction_info": {"source": "custom"},
-            "contexts": contexts,
-            "start_timestamp": convert_from_otel_timestamp(span.start_time),
-            "timestamp": convert_from_otel_timestamp(span.end_time),
-        }  # type: Event
+        event = self._span_to_json(span)
```

Comment:
this doesn't work I think, because the transaction doesn't have stuff like `description`. In the old version, we always serialized transaction separately without calling `super`

Reply:
This is a mess rn, there's a `to_json()` on `Transaction` that calls `super()` but it's not used in `finish()` where we actually serialize 🙃 

Once we drop transactions we'll need to do this.

Ok then for now I move the common stuff (timestamps + measurements + tags) that can be on both transactions and spans to some common util function.

Reply:
fwiw, relay will mostly just accept random fields silently and drop them (we can check) but I'm not sure we want to do that :)

Reply:
updated